diff -ur a/wallutils.sh b/wallutils.sh
--- a/wallutils.sh	2020-07-20 15:51:54.421061526 +0200
+++ b/wallutils.sh	2020-07-22 21:27:43.691146958 +0200
@@ -99,12 +99,23 @@
     if ! iconf -i nologo; then
         instantoverlay
         imgresize overlay.png "$RESOLUTION"
-        convert wall.png -channel RGB -negate invert.png
-        convert overlay.png invert.png -compose Multiply -composite out.png
-        composite out.png wall.png instantwallpaper.png
-        rm wall.png
-        rm invert.png
-        rm out.png
+
+		# create mask from overlay
+        convert overlay.png  -alpha extract mask.png
+        # cut the image with the mask
+        composite -compose CopyOpacity mask.png wall.png cut.png
+        # Convert to black and white the cut
+        convert cut.png -colorspace Gray blackandwhite.png
+        # Negate the black and white cut
+        convert blackandwhite.png -channel RGB -negate invert.png
+        # draw the computed overlay on top of the background
+        convert wall.png invert.png -gravity center -composite instantwallpaper.png
+
+        # rm wall.png
+		# rm mask.png
+		# rm cut.png
+		# rm blackandwhite.png
+		# rm invert.png
     else
         echo "logo disabled"
         mv wall.png instantwallpaper.png
